fix QR and new connect_to_obs

nglmercer opened the pull request:
def connect_to_obs(self, host: str, port: int, password: str) -> None:
- Ideas inicial de como los 3 valores a recibir para conectar al OBS
- Arreglo del problema con el QR que recibía un objeto en vez de la imagen

nglmercer committed:
fix QR and new connect_to_obs

def connect_to_obs(self, host: str, port: int, password: str) -> None:
- Ideas inicial de como los 3 valores a recibir para conectar al obs
-Arreglo del problema con el qr que recibia un objeto envez de la imagen

GatoArtStudios commented:
Esta mal la logica/estructurado, y que el codigo que propone:
```python
class API_OBS:
    def __init__(self) -> None:
        self.connection = False
        self.client = None

    def connect_to_obs(self, host: str, port: int, password: str) -> None:
        """
        Connect to OBS using the provided host, port, and password.
        """
        try:
            self.client = obsws(host=host, port=port, password=password)
            self.client.connect()
            self.connection = True
        except Exception as e:
            print('Error al conectar a OBS:', e)

    def get_version(self):
        data = self.client.call(requests.GetVersion())

        elif action == 'record-toggle-pause':
            self.client.call(requests.ToggleRecordPause())

apiObs = API_OBS()
apiObs.connect_to_obs('localhost', 4455, '123456')
```
El codigo con una mejor estructura y que nos sirve para un objeto por que precisamente es un objeto seria la siguiente:
```python
class API_OBS:
    def __init__(self) -> None:
        self.connection = False
        self.client = None
        self.obs_default = {
            'host': 'localhost',
            'port': 4455,
            'passd': '123456'
        }

    def connect_to_obs(self) -> None: # Llama el metodo para poder controlar el momento en que debe conectarce
        """
        Connect to OBS using the provided host, port, and password.
        """
        try:
            self.client = obsws(host=self.obs_default['host'], port=self.obs_default['port'], password=self.obs_default['passd'])
            self.client.connect()
            self.connection = True
        except Exception as e:
            print('Error al conectar a OBS:', e)

    def get_version(self):
        data = self.client.call(requests.GetVersion())
        # Codigo...
        elif action == 'record-toggle-pause':
            self.client.call(requests.ToggleRecordPause())

apiObs = API_OBS()
# apiObs.connect_to_obs('localhost', 4455, '123456') # este metodo se llamaria desde otro modulo, etc
``` 
Por que de esta forma, de esta forma podemos controlar en todo momento sobre la conexion al obs cuando necesitemos, necesitamos almacenar las credenciales por defecto para el obs como un atributo del objeto para asi poder modificalos facilmente desde otro modulo o el hilo principal, no dara mas control tanto como para conectar como para poder modificar las credenciales que de esta forma son un atributo del objeto y no hay que pasarle las credenciales a cada rato, da mas control y mejor control en cuando editar el objeto que es la intencion de un objeto, con el tema de que el usuario pueda editar los datos desde la ventana de flet de esta forma nos dara  mas control, y como es un modulo, el metodo `connect_to_obs()` hay que llamarlo desde fuera, esto nos da mucho mas control sobre cuando conectar, tambien se puede agregar otro metodo para desconectar en caso de que el cliente cambie las credenciales.

nglmercer committed:
OBS connect

conexion a OBS mediante un formulario desde el front-end para conectarse con parametros predefinidos o pesonalizados
